vms/administrator/tests/test_report.py

```diff
@@ -52,176 +54,101 @@ def login_admin(self):
+        vol = create_volunteer()
+        register_past_event_utility()
+        register_past_job_utility()
+        shift = register_past_shift_utility()
+        start=datetime.time(hour=10, minute=0)
+        end=datetime.time(hour=11, minute=0)
+        logged_shift = log_hours_with_details(vol, shift, start, end)
+        create_report_with_details(vol, logged_shift)
         report_page = self.report_page
-        # Register dataset
-        org = create_organization_with_details('organization-one')
-        volunteer = create_volunteer()
-        volunteer.organization = org
-        volunteer.save()
-
-        # Create Event
-        event = ['Hackathon', '2050-05-24', '2050-05-28']
-        created_event = create_event_with_details(event)
-
-        # Create Job
-        job = ['Developer', '2050-05-24', '2050-05-28', '', created_event]
-        created_job = create_job_with_details(job)
-
-        # Create Shift
-        shift = ['2050-05-24', '09:00', '15:00', '10', created_job]
-        created_shift = create_shift_with_details(shift)
-
-        # Shift is assigned to volunteer-one, but hours have not been logged
-        report_page.go_to_admin_report()
-        register_volunteer_for_shift_utility(created_shift, volunteer)
         report_page.get_page(self.live_server_url, PageUrls.administrator_report_page)
-        # Check admin report with null fields, should not return the above shift
-        # Using multiple tries so to cater to late loading of page.
-        for _ in range(3):
-            try:
-                report_page.fill_report_form(['', '', '', '', ''])
-                break
-            except NoSuchElementException:
-                pass
-        self.assertEqual(report_page.get_alert_box_text(), report_page.no_results_message)
-
-    @staticmethod
-    def register_dataset(parameters):
-        # Register dataset
-        # Register dataset
-        volunteer = create_volunteer_with_details_dynamic_password(parameters['volunteer'])
-        volunteer.organization = parameters['org']
-        volunteer.save()
-
-        # Create Event
-        event = parameters['event']
-        created_event = create_event_with_details(event)
-
-        # Create Job
-        job = parameters['job'] + [created_event]
-        created_job = create_job_with_details(job)
-
-        # Create Shift
-        shift = parameters['shift'] + [created_job]
-        created_shift = create_shift_with_details(shift)
-
-        # Create VolunteerShift
-        log_hours_with_details(volunteer, created_shift, parameters['vshift'][0], parameters['vshift'][0])
-
-    def create_dataset(self):
-        orgs = Organization.create_multiple_organizations(4)
-
-        test_data = open('test_data.json').read()
-        parameters = json.loads(test_data)
-
-        parameters[0]["org"] = orgs[0]
-        self.register_dataset(parameters[0])
+        self.assertEqual(report_page.element_by_xpath(self.elements.REJECT_REPORT).text, 'Reject')
```

Comment:
Make a function to return this `element_by_xpath(self.elements.REJECT_REPORT).text`.